phnt/include/ntmmapi.h

```diff
@@ -729,9 +729,9 @@ NTSTATUS
 NTAPI
 NtReadVirtualMemory(
     _In_ HANDLE ProcessHandle,
-    _In_opt_ PVOID BaseAddress,
-    _Out_writes_bytes_(BufferSize) PVOID Buffer,
-    _In_ SIZE_T BufferSize,
+    _In_ PVOID BaseAddress,
```

Comment:
BaseAddress is `_In_opt_` because NULL (zero) is a valid address and `_In_` incorrectly throws warnings

Reply:
Committed, all `BaseAddress` are reverted to `_In_opt_`.